Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
```

Comment:
In the 'dev_server' command, the use of 'go run' on external packages can lead to issues if dependencies change or the package has breaking changes. Consider replacing this with a more stable build process, such as 'go install', to improve reproducibility and keep the build predictable.